recipes/mumax3c/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "mumax3c" %}
+{% set version = "0.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 156b5b36ea89473f699a0d35ea4d4a8c030d26a4839627e824d0a69d0471c39c
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python >=3.8
+    - pip
+  run:
+    - python >=3.8
+    - micromagneticmodel >=0.62.0
+    - micromagnetictests >=0.62.0
+    - scipy
+    - ubermagtable >=0.61.1
+    - ubermagutil >=0.63.0
+
+test:
+  imports:
+    - mumax3c
+  commands:
+    - pip check
+    # running unittests would require a GPU and a separate Mumax3 installation
+  requires:
+    - pip
+
+about:
+  home: http://ubermag.github.io/
+  summary: Python interface to mumax3 integrated into Jupyter notebook.
+  license: BSD-3-Clause
+  license_family: BSD
+  license_file: LICENSE
+  doc_url: https://ubermag.github.io/documentation/mumax3c.html
```

Comment:
This url returns 404. Could you please fix?

Reply:
The website is now online. (We initially intended to make it public once the package is available on conda-forge)